conf/messages.cy

```diff
@@ -53,7 +53,8 @@ lsp.onThreshold.p1 = Rydych wedi cyrraedd y trothwy ar gyfer cosbau ariannol.
 agent.lsp.onThreshold.p1 = Mae eich cleient wedi cyrraedd y trothwy ar gyfer cosbau ariannol.
 lsp.onThreshold.p2 = Mae hyn yn golygu bod yn rhaid i chi dalu cosb o £{0} bob tro yr ydych yn cyflwyno Ffurflen TAW yn hwyr, hyd nes yr ydym yn tynnu’ch pwyntiau cosb.
 agent.lsp.onThreshold.p2 = This means they have to pay a £{0} penalty every time they submit a VAT Return late, until we remove their points.
-lsp.onThreshold.link = Camau i’w cymryd cyn y gall CThEM dynnu’ch pwyntiau
+lsp.onThreshold.p3 = Earliest dates your points could be removed:
+lsp.onThreshold.link = Actions to take to get your points removed by {0}
```

Comment:
the message would be 'Actions to take to get your points removed' instead, 'by {0}' could be deleted.

Reply:
ditto as above